src/lib/components/comparisonTable/index.stories.mdx

```diff
@@ -27,17 +27,22 @@ The Comparison Table component provides an easy way to compare vast amounts of i
-| stickyHeaderTopOffset | number              | Y-offset of the sticky header row                          | 0px           | false    |
-| growContent           | boolean             | Makes the content area grow to fill the available space    | false         | false    |
+| attribute             | unit                                      | description                                                         | default value  | required |
+| --------------------- | ----------------------------------------- | ------------------------------------------------------------------- | -------------- | -------- |
+| headers               | [Header[]](#header)                       | The structure of the table                                          | n/a            | true     |
+| data                  | array                                     | The title text that needs to be displayed                           | n/a            | true     |
+| hideDetails           | boolean                                   | Hide table groups that do not have the `default` attribute          | false          | false    |
+| hideDetailsCaption    | string                                    | Caption of the button to hide the details                           | 'Hide details' | false    |
+| showDetailsCaption    | string                                    | Caption of the button to show the details                           | 'Show details' | false    |
+| hideScrollBars        | boolean                                   | Hide scroll bars                                                    | false          | false    |
+| hideScrollBarsMobile  | boolean                                   | Hide scroll bars on mobile only                                     | false          | false    |
+| collapsibleSections   | boolean                                   | Make table groups with a label collapsible                          | false          | false    |
+| cellWidth             | number                                    | Width of a table content cell                                       | 211px          | false    |
+| firstColumnWidth      | number                                    | Width of the first column of the table                              | 212px          | false    |
+| stickyHeaderTopOffset | number                                    | Y-offset of the sticky header row                                   | 0px            | false    |
+| growContent           | boolean                                   | Makes the content area grow to fill the available space             | false          | false    |
+| onSelectionChanged    | (number) => void                          | Callback to be called when the selected tab index changes on mobile | n/a            | false    |
+| classNameOverrides    | [ClassNameOverrides](#classnameoverrides) | "className" props to be passed down to various internal components  | n/a            | false    |
```

Comment:
Eventually would be cool to move this to JSX stories but we can wait for a major refactor.

Reply:
Yeah true, didn't think about that!